src/frameworks/compose/discover/frameworkMatcher.ts

```diff
@@ -0,0 +1,95 @@
+import { FrameworkSpec, FileSystem } from "./types";
+
+/**
+ *
+ */
+export function filterFrameworksWithDependencies(
+  allFrameworkSpecs: FrameworkSpec[],
+  dependencies: Record<string, string>
+): FrameworkSpec[] {
+  return allFrameworkSpecs.filter((framework) => {
+    return framework.requiredDependencies.every((dep) => {
+      return dep["name"] in dependencies;
```

Comment:
nit* prefer `dep.name` over `dep["name"]`

Reply:
See TS styleguide for "optimization-compatibility-for-property-access" (I think that's the relevant piece)